compose/services.yml

```diff
@@ -113,20 +114,21 @@ services:
     extends:
       file: ./common.yml
       service: system
-    image: balena/open-balena-db:${OPENBALENA_DB_VERSION_TAG:-master}
+    image: balena/open-balena-db:${OPENBALENA_DB_VERSION_TAG}
     volumes:
       - db:/var/lib/postgresql/data
 
   s3:
     extends:
       file: ./common.yml
       service: component
-    image: balena/open-balena-s3:${OPENBALENA_S3_VERSION_TAG:-master}
+    image: balena/open-balena-s3:${OPENBALENA_S3_VERSION_TAG}
     volumes:
       - s3:/export
     environment:
-      S3_MINIO_ACCESS_KEY: abcdef1234
-      S3_MINIO_SECRET_KEY: "1234567890"
+      S3_MINIO_ACCESS_KEY: ${OPENBALENA_S3_ACCESS_KEY}
+      S3_MINIO_SECRET_KEY: ${OPENBALENA_S3_SECRET_KEY}
+      BUCKETS: ${OPENBALENA_S3_BUCKETS}
```

Comment:
Does Minio automatically create buckets? That would be awesome.

Reply:
@dfunckt It doesn't, but I have added the functionality to our build so that it does :)